mne/viz/tests/test_epochs.py

```diff
@@ -137,6 +137,7 @@ def test_plot_epochs_image():
     """Test plotting of epochs image."""
     epochs = _get_epochs()
     epochs.plot_image(picks=[1, 2])
+    epochs.plot_image(picks='mag')
```

Comment:
We could have a test where we count the length of what epochs.plot_image returns too.

Reply:
I mean, 5 figs, <5 figs, >5 figs.

Reply:
see my (monster) comment with the table.  I'm proposing that we get rid of the magic number 5 and give people what they ask for, but make it harder for them to ask for lots of figs (i.e., they only can get lots if they list channel {names|indices} individually, or pass a group_by dict with lots and lots of keys).  We can (should) probably add a note to the docstring warning people not  to do too many at once, and if they want to view lots of epochs imagemaps, they should consider the topo version that allows them to click through each sensor one at a time.

Reply:
+1